internal/controller/pipelinerollout_controller.go

```diff
@@ -467,21 +466,18 @@ func (r *PipelineRolloutReconciler) processExistingPipeline(ctx context.Context,
 
 	// if not, should we set one?
 	if !inProgressStrategySet {
-		// inProgressStrategy is used for PPND and Progressive strategies (i.e. any strategies which require multiple reconciliations to perform)
```

Comment:
you don't think we need a comment like this? (although maybe above)

Reply:
I feel this comment is a little bit irrelevant and redundant, so removed it. And the code above seems well documented to me. :)